activestorage/lib/active_storage/validations/size_validator.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module ActiveStorage
+  module Validations
+    class SizeValidator < ActiveModel::EachValidator
+      delegate :number_to_human_size, to: ActiveSupport::NumberHelper
+
+      AVAILABLE_CHECKS = [:less_than, :less_than_or_equal_to, :greater_than, :greater_than_or_equal_to, :between]
+
+      def check_validity!
+        unless (AVAILABLE_CHECKS).any? { |argument| options.has_key?(argument) }
+          raise ArgumentError, "You must pass either :less_than, :greater_than, or :between to the validator"
```

Comment:
Should this include all of the `AVAILABLE_CHECKS`? Or more realistically just use these as examples and keep the error message as "You must pass a validation check to the validator."